packages/xo-web/src/common/intl/messages.js

```diff
@@ -1537,9 +1537,9 @@ const messages = {
   autoPowerOn: 'Auto power on',
   protectFromDeletion: 'Protect from accidental deletion',
   protectFromShutdown: 'Protect from accidental shutdown',
-  blockMigration: 'Block migration',
-  unblockMigrationTitle: 'Unblock migration',
-  unblockMigrationConfirm: 'Are you sure you want to unblock migration for this VM?',
+  blockMigration: 'Disable migration',
+  unblockMigrationTitle: 'Enable migration',
+  unblockMigrationConfirm: 'Are you sure you want to enable migration for this VM?',
```

Comment:
This one too: https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/intl/messages.js#L2026